lesson02/cardDeck.js

```diff
@@ -0,0 +1,87 @@
+// L2C5 CardDeck
+
+// Write a program called cardDeck.js. Create the code in codpen.io and 
+// then copy that code into a document and add to your GitHub in the 
+// Lesson02 directory.
+
+// Write a program that prints out the values of a deck of cards.
+
+// Save your work to the Lesson02 directory, on your lesson2-challenge5 
+// Git branch.
+
+
+let card_values, card_suits, text, csLen, cvLen, i, j;
+card_values = ["Ace", "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine", "Ten", "Jack", "Queen", "King"];
+card_suits = ['Hearts', 'Diamonds', 'Clubs', 'Spades'];
+csLen = card_suits.length;
+cvLen = card_values.length;
+for (i = 0; i < cvLen; i++) {
+    console.log(card_values[i]);
+    for (j = 0; j < csLen; j++) {
+      console.log(' of ' + card_suits[j]);
```

Comment:
Great work here. Very tiny detail, but would recommend printing the card and suit on same line. So `console.log(card_values[i] + ' of ' + card_suits[j]`